pages/open_cas_linux/guide_running.md

```diff
@@ -0,0 +1,213 @@
+---
+title: Open CAS Linux - Admin Guide
+last_updated: April 29, 2019
+toc: true
+permalink: guide_running.html
+---
+
+Running Open CAS Linux
+==================
+
+This chapter describes how to perform typical administrative activities with
+Open CAS Linux.
+
+Initial configuration of Cache instances
+----------------------------------------
+
+Once Open CAS Linux is installed, but no cache instances are yet configured and
+running, the user can easily configure and start cache instances. Refer to the
+guidance in [**Configuring CAS**](/guide_configuring.html) for configuring the file *utils/opencas.conf*, then
+follow these steps:
+
+1. Edit and configure caches and cores as desired via the
+*opencas.conf* file.
+
+2. Execute the following:
+
+>   \# utils/casctl init
```

Comment:
The `casctl` is now installed by `make install` so there is no need to point to `utils/` directory.